packages/tfjs-node-helpers/src/testing/confusion-matrix.ts

```diff
@@ -1,28 +1,24 @@
+import { math, Tensor } from '@tensorflow/tfjs-node';
+
 export class ConfusionMatrix {
   public tp: number;
   public tn: number;
   public fp: number;
   public fn: number;
 
-  constructor(trueValues: Float32Array, predictedValues: Float32Array) {
-    this.tp = 0;
-    this.tn = 0;
-    this.fp = 0;
-    this.fn = 0;
+  constructor(trueValues: Tensor, predictedValues: Tensor) {
+    const trueY = trueValues.as1D();
+    const predY = predictedValues.as1D();
+    const confusionMatrix = math.confusionMatrix(trueY, predY, 2);
+    const [[tn, fp], [fn, tp]] = confusionMatrix.arraySync();
```

Comment:
@VladimirGlinskii It would be great if we could use the async variant of this call. Maybe we could plan this refactoring for the future.

Reply:
@dmitryusenko Let's do this after we implement multiclass classification

Reply:
@VladimirGlinskii Sounds like a good plan.